templates/internet-of-things/smart-home.html

```diff
@@ -0,0 +1,177 @@
+      {{ image (
+        url="https://assets.ubuntu.com/v1/a43d61ae-update+control.svg",
+        alt="",
+        width="150",
+        height="150",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h3>Continuous security updates</h3>
+      <p>As soon as vulnerabilities become known, security patches are delivered to each device in your fleet. Our Extended Security Maintenance offering for Ubuntu Core is active for up to ten years. You don't have to worry about CVEs, we keep your security debt at zero.</p>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h3>Fleet management through the snap store</h3>
+      <p>Get fine-grained control on OTAs. Roll out delta updates gradually, with the confidence of automatic rollback. Know that each version of each application on every device has been tested together with <a href="/core/docs/reference/assertions/validation-set">validation sets</a>.</p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/da83f401-4+-+Register+to+the+snap+store+and+distribute+your+application+to+RasPi+users+everywhere.svg",
+        alt="",
+        width="232",
+        height="150",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>How to develop your open source smart home application</h2>
+
+    <ol class="p-stepped-list--detailed">
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Get started with any development kit</h3>
+
+        <div class="p-stepped-list__content">
+          <p>Ubuntu supports a large <a href="/certified">library of hardware</a> out of the box, and Canonical provides board support as a service. This means you can start your development before your hardware is finalised, with the confidence that your application will run smoothly on the final product.</p>
+        </div>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Find the building blocks of your application in the Snap Store</h3>
+
+        <div class="p-stepped-list__content">
+          <p>The <a href="https://snapcraft.io/search?category=devices-and-iot">Snap Store</a> includes a huge collection of open source projects that can be seamlessly integrated into your application. Each snap comes with the same level of OTA and revision control you have for your own application.</p>
+
+
+          {{ image (
+            url="https://assets.ubuntu.com/v1/d22c1230-snap-store-iot-devices.png",
+            alt="",
+            width="636",
+            height="346",
+            hi_def=True,
+            loading="lazy"
```

Comment:
```suggestion
            loading="lazy",
            attrs={"class": "u-hide--small"}
```
It is difficult to see the image on small screens, so we might as well hide it